00_Base/src/util/model/CountryCode.ts

```diff
@@ -0,0 +1,267 @@
+import {assert} from "./global";
+
+export type CountryCode = typeof COUNTRY_CODES[number];
+
+export function isCountryCode(value: string): value is CountryCode {
+    return (COUNTRY_CODES as readonly string[]).includes(value); // TODO: optimize
+}
+
+export function countryCode(value: string): CountryCode {
+    assert(isCountryCode(value), `Invalid country code: ${value}`);
+    return value;
+}
+
+// ISO 3166-1 alpha-2 country codes
+export const COUNTRY_CODES = [
```

Comment:
Should we limit the country codes to only those that we plan to support for now?